modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/rest/api/server/application/management/v1/adaptive-metadata.json

```diff
@@ -1,3 +1,3 @@
 }
```

Comment:
shall we add a newline

Reply:
Fixed with https://github.com/wso2/product-is/pull/14903